### ISSUE TITLE
Does mongoshake support MongoDB version 8.0.1?

### ISSUE THREAD

**Issue by qwzhou89** [alibaba/MongoShake#889]
Can mongoshake be used for migrating MongoDB from version 5.0.5 (source) to version 8.0.1 (target)?

**Comment by zhongli-james** [comment]
Yes, in principle, mongoshake does not use features that are not forward-compatible. After Alibaba Cloud officially releases MongoDB8.0 version recently, we will also use mongoshake for related tests.

PS: Recently, new CVEs has been officially released. It is recommended that you upgrade the version of the instance from 8.0.1 to 8.0.4+
https://www.mongodb.com/resources/products/mongodb-security-bulletins

**Comment by zhongli-james** [comment]
Clear historical issues that have not been closed. If anyone has the same problem, please raise a new issue and link it. Thank you.
